CONTRIBUTING.rst

```diff
@@ -487,3 +487,13 @@ Before we can accept your pull requests you'll need to sign a Contributor Licens
 - **If you work for a company that wants to allow you to contribute your work**, then you'll need to sign a `corporate CLA <https://developers.google.com/open-source/cla/corporate>`__.
 
 You can sign these electronically (just scroll to the bottom). After that, we'll be able to accept your pull requests.
+
+*******************************
+Helpful Pull Request Guidelines
+*******************************
```

Comment:
Right. Though maybe "please, pretty please, pretty pretty please read". Also that isn't the correct syntax for an RST link, though you can get that right in the code?